packages/libs/react-components/src/components/Grid/GridItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { IThemeColors, styled } from './../../styles';
+
+import React, { FC, ReactNode } from 'react';
+
+const BaseItem = styled('div', {
+  gridColumnStart: 'auto',
+});
+
+export type IColSpan = number | 'auto';
+
+export interface IGridItemProps {
+  children?: ReactNode;
+  colStart?: IColSpan;
+  colEnd?: IColSpan;
+  rowStart?: IColSpan;
+  rowEnd?: IColSpan;
+  area?: string;
+  bg?: IThemeColors;
+}
+
+const checkSpan = (
```

Comment:
What is this checking? Should it be validating that the colEnd is past colStart?

Reply:
we can add that check. 
But is is really just creating that string.